src/AppBundle/Controller/TaskController.php

```diff
@@ -45,13 +45,15 @@ public function createAction(Request $request)
     /**
      * @Route("/tasks/{id}/edit", name="task_edit")
      */
-    public function editAction(Task $task, Request $request)
+    public function editAction(int $id, Request $request)
```

Comment:
Because with task there was a bug and the request returned an error... Replace the task by the id works in that case but perhaps there was an other solution ?